CS/DataContext_Ex/MainWindow.xaml

```diff
@@ -1,96 +1,117 @@
-<Window x:Class="DataContext_Ex.MainWindow" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:dxd="http://schemas.devexpress.com/winfx/2008/xaml/docking" xmlns:local="clr-namespace:DataContext_Ex" Title="MainWindow" Height="449" Width="525">
+<Window
+    x:Class="DataContext_Ex.MainWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:dxd="http://schemas.devexpress.com/winfx/2008/xaml/docking"
+    xmlns:local="clr-namespace:DataContext_Ex"
+    Title="MainWindow"
+    Width="525"
+    Height="449">
     <dxd:DockLayoutManager x:Name="dm1">
-        <!--Set DataContext for DockLayoutManager. By default, this DataContext will be inherited by all child elements-->
+        <!--  Set DataContext for DockLayoutManager. By default, this DataContext will be inherited by all child elements  -->

```

Comment:
Please update this comment without using "By default".

Reply:
```suggestion
        <!--  Set DataContext for DockLayoutManager. DockLayoutManager's child elements inherit this DataContext -->

```